sdcm/tester.py

```diff
@@ -216,6 +216,15 @@ def setUp(self):
             self.db_cluster.wait_for_init()
         if self.cs_db_cluster:
             self.cs_db_cluster.wait_for_init()
+
+        self.log.info('change RF of system_auth to 3')
+        # change RF of system_auth to 3
+        node = self.db_cluster.nodes[0]
+        session = self.cql_connection_patient(node)
+        session.execute("ALTER KEYSPACE system_auth WITH replication = {'class': 'org.apache.cassandra.locator.SimpleStrategy', 'replication_factor':3};")
```

Comment:
there no way to config it straight from configuration (in scylla.yaml) ?

Reply:
No, system_auth is a system keyspace, the RF can only be changed by an ALERT query.